lib/rubygems/resolver.rb

```diff
@@ -38,8 +38,6 @@ class Gem::Resolver
   ##
   # List of dependencies that could not be found in the configured sources.
```

Comment:
This comment should also be removed, since it was the documentation for `#missing`